src/qw/changes.py

```diff
@@ -14,43 +13,141 @@
 class ChangeHandler:
     """Allow user interaction to manage changes between local and remote design stage data."""
 
+    class DiffElement:
+        """A potential update to the local store."""
```

Comment:
ooh fun

Reply:
Yeah, I wanted to be able to separate out the gathering of the diffs from the acceptance of them, so that required this class.